.github/workflows/.env/nightly-tests/min_versions.env

```diff
@@ -1,7 +1,7 @@
-node_version='v1.28.6@sha256:b7e1cf6b2b729f604133c667a6be8aab6f4dde5bb042c1891ae248d9154f665b'
-kubectl_version='v1.28.15'
-kind_version='v0.21.0'
-helm_version='v3.13.3'
+node_version='v1.29.2@sha256:51a1434a5397193442f0be2a297b488b6c919ce8a3931be0ce822606ea5ca245'
```

Comment:
Let's make sure to ping the docs team to ensure they update https://docs.solo.io/gloo-edge/latest/reference/support/#supported-versions to reflect this as well

Reply:
@solo-io/solo-docs Please notice this the changes in the min/max version in this PR

Reply:
Thanks for the tag, I updated in this doc version fixes PR: https://github.com/solo-io/gloo/pull/10909